build.gradle.kts

```diff
@@ -72,20 +52,20 @@ configurations.all {
 // Configure Gradle IntelliJ Plugin
 // Read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
 intellij {
-  version.set("2022.1.4")
+  version.set("2022.3.1")
```

Comment:
Is this change mandatory?

Reply:
Compatibility range of ide is same for both . Its didn't affect plugin but reduced deprecated api warnings in IntelliJ verifier after plugin built

Reply:
for making sure plugin is compatible for upcoming updates from Jetbrains

Reply:
https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#intellij-platform-based-products-of-recent-ide-versions

Reply:
https://plugins.jetbrains.com/docs/intellij/api-changes-list-2023.html